.github/workflows/full-analysis.yml

```
@@ -20,6 +20,7 @@ jobs:
       # full-live-analysis.yml acquires live evidence from providers.
       ALLOW_STATIC_RECOVERY_MODE: "true"
       STATIC_RECOVERY_REASON: "Full Analysis CI reproducibility check"
+      MORSKAMARY_CUMULATIVE_DATABASE_DIR: ${{ runner.temp }}/full-analysis-retained-inputs/cumulative_database
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)